config/admin-authz-configuration.yaml

```diff
@@ -0,0 +1,25 @@
+---
+# This file contains the role mapping for the admin API based on the HTTP methods.
+# Each HTTP method allows configuring an arbitrary amount of roles that authorize requests to the API.
+# Each of the role mapping must correspond to an existing Rover group 
+# (https://rover.redhat.com/groups/) in order to grant access to the HTTP method containing that role
+# Configuration presented below is only used for testing purposes. The actual configuration deployed in 
+# production and stage environments will be provided in the saas template in app-interface
+- method: GET
+  roles:
+    - "kas-fleet-manager-admin-full-stage"
+    - "kas-fleet-manager-admin-read-stage"
```

Comment:
@R-Lawton Do you agree with role name and formate?

Reply:
Yeah the kas fleet manager ones look good, for the connectors why don't we differentiate the environment?

Reply:
there is more to do on the connectors side (including the roles names), but I guess I can change it as well